lib/cli.js

```diff
@@ -309,9 +311,17 @@ internals.options = function () {
         var transform = require(Path.resolve(options.transform));
 
         Hoek.assert(Array.isArray(transform), 'transform module must export an array of objects {ext: ".js", transform: null or function (content, filename)}');
-
         options.transform = transform;
+
+        var includes = 'js|' + transform.map(internals.mapTransform).join('|');
```

Comment:
This will create a wrong regex ( `.(js|)$` ) if the transform module exports an empty array. Also, the dot should be escaped, below and above too.